.github/workflows/build-docker.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v4
         with:
-          images: ${{ secrets.DOCKERHUB_USERNAME }}/dao-dashboard
+          images: api3/dao-dashboard
```

Comment:
Shall this be also replaced on line 30? Why is it actually replaced?

Reply:
Aaron created a new user, `api3daodashboard`, specifically for this repo ([see here](https://github.com/api3dao/api3-dao-dashboard/pull/360#issuecomment-1325386654)) hence the image is `api3` to correspond with the repo and the username is passed as a secret (could be hardcoded as well, but no need).